iocage/Release.py

```diff
@@ -215,7 +215,12 @@ class ReleaseGenerator(ReleaseResource):
         "sendmail_submit_enable": False,
         "sendmail_msp_queue_enable": False,
         "sendmail_outbound_enable": False,
-        "syslogd_flags": "-ss"
+        "daily_clean_hoststat_enable": False,
+        "daily_status_mail_rejects_enable": False,
+        "daily_status_include_submit_mailq": False,
+        "daily_submit_queuerun": False,
+        "syslogd_flags": "-ss",
```

Comment:
you could place syslogd_flags to the end, so that the line does not change

Reply:
that would make sense, esp given that the cron line actually groups *with* this